images/ceph/Dockerfile

```diff
@@ -20,7 +20,11 @@ ARG TINI_VERSION
 
 # Run tini as PID 1 and avoid signal handling issues
 RUN curl --fail -sSL -o /tini https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-${ARCH} && \
-    chmod +x /tini
+    chmod +x /tini && \
+    curl -LJO https://github.com/ceph/persistent-volume-migrator/releases/download/v0.1.0-alpha/pv-migrator && \
```

Comment:
We need to build for different architectures, do we? This will fail on arm64 right now.

Reply:
Can we add a conditional for amd64 in the dockerfile? Or for now we just pull an invalid image for arm64, then fix it later if needed?

Reply:
Per discussion, going with what we have to get the testing going...